withings_sync/sync.py

```diff
@@ -470,6 +472,7 @@ def sync():
                     logging.info(
                         "Fit file with blood pressure information uploaded to Garmin Connect"
                     )
+                    # Save this sync so we don't re-download the same data again (if no range has been specified)
```

Comment:
There is no longer need for this comment here.

Reply:
removed, thanks!